src/Library/demos/Box/main.blp

```diff
@@ -0,0 +1,109 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Clamp {
+  Box {
+    orientation: vertical;
+
+    Label {
+      label: "Box";
+      margin-top: 12;
+      margin-bottom: 12;
+
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: "A widget that arranges its child widgets into a single row or column";
+      margin-bottom: 18;
+    }
+
+    Box {
+      orientation: vertical;
+
+      Box {
+        hexpand: true;
+        orientation: vertical;
+        spacing: 12;
+
+        Box {
+          halign: center;
+          spacing: 18;
+          Button button_add{
+            label: "Add Item";
+            styles [
+              "suggested-action",
```

Comment:
I would drop `suggested-action` - it's not more or less important than the second